### DISCUSSION TITLE
case insensitive sorting for extension

### DISCUSSION THREAD

**Discussion by ciros88** [doublecmd/doublecmd#987] [General]
it is possible to apply the sorting case sensivity to extensions also?

often this scenario happens:
i have a directory with files ordered by extension, but some of these are capital. this is an example:
![image](https://github.com/doublecmd/doublecmd/assets/84340161/9c603839-8fab-462f-b1b3-7a295dda96d7)
as u can see in both panels, the case insensitivty ordering is applied on filenames only. there is something i can do? thanks

**Comment by alexx2000** [comment]
DC use same sortings rules for name and extension.

**Comment by ciros88** [comment]
So why i have the that result in the screen? Seems that extension sorting puts the capital MOD and XM first

**Reply by decart** [reply]
Hi. It looks like the “Files views ⇒ Sorting ⇒ Case sensitivity” setting is sets to “according to locale settings (aAbBcC)”. Anyway, I was able to replicate this behavior by enabling this setting.

**Reply by ciros88** [reply]
its look like the extensions is AaBbCc (instead of case insensitive) while filenames is aAbBcC. these are my settings:

![image](https://github.com/doublecmd/doublecmd/assets/84340161/c8059b8c-8e8a-43d0-9bcb-28388e66d7a6)

**Reply by decart** [reply]
Finally, I found the reason of that behavior. Try to change “Sort method” to “Alphabetical, considering accents”. But what is this: bug or feature?))

**Reply by ciros88** [reply]
yeah, with this setting now extension sorting seems right. this sounds me more like a bug. i mean, i don't see the point to have filenames case insensitive sorting (by default) but not extension. @alexx2000 what do u think about?

**Reply by alexx2000** [reply]
I will look why sorting is wrong in this case.

**Comment by alexx2000** [comment]
Fixed (commit [48865cf](https://github.com/doublecmd/doublecmd/commit/48865cfec45df49b694a41acd6378f4fa157b27d)).
